### ISSUE TITLE
The navigation order is logical and intuitive.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#124]
When users press the "Back to Top" button, the page scrolls visually to the top, but the keyboard focus remains on the button. Ensure that focus is programmatically moved to the top of the page.

On mobile, when VoiceOver users activate the "More Information (+)" button, the image gallery modal appears but focus does not move to the modal immediately. Instead, users can still navigate the main page content, and they cannot easily move out of the modal without closing it because it is implemented using a <div> element. Ensure that focus moves to the modal when it opens and remains trapped within it until the modal is dismissed. When the modal is closed using an interactive <button> element, such as the "Close (X)" button, focus should return to the "More Information (+)" button that triggered it.
